test/runtests.jl

```diff
@@ -10,19 +10,19 @@ solvers = [
            ODE.ode2_midpoint,
            ODE.ode2_heun,
            ODE.ode4,
-           ODE.ode4ms,
-           ODE.ode5ms,
+           # Ode.ode4ms,
```

Comment:
Sorry, I just didn't have time to implement them with iterators.  I was planning to initially so I removed the methods to add them later.  I should re-include the old implementation and leave it there.